src/components/common.ts

```diff
@@ -1,17 +1,10 @@
 import { Store } from "./NostoProduct/store"
 
-export function syncPrices(element: HTMLElement, setter: Store["setPrices"]) {
-  const price = element.getAttribute("n-price")
-  if (price) {
-    const listPrice = element.getAttribute("n-list-price")
-    setter(price, listPrice || undefined)
-  }
-}
-
-export function syncImages(element: HTMLElement, setter: Store["setImages"]) {
-  const image = element.getAttribute("n-img")
-  if (image) {
-    const altImage = element.getAttribute("n-alt-img")
-    setter(image, altImage || undefined)
-  }
+export function syncSkuData(element: HTMLElement, setter: Store["setSkuData"]) {
+  setter({
```

Comment:
consider restricting this logic to trigger setter only for Truthy values? Values are already undefined in the store. Now, we are invoking the setting again with undefined values which triggers the listeners on these fields

Reply:
not true
please check the implementation of setSkuData

Reply:
You are right.